packages/node/src/node.ts

```diff
@@ -61,83 +77,51 @@ export default class Node {
     this.incoming.emit(event, req);
   }
 
-  get address() {
-    return this.signer.address;
-  }
-
-  /**
-   * Sends a message to another Node. It also auto-includes the from field
-   * in the message.
-   * @param peerAddress The peer to whom the message is being sent.
-   * @param msg The message that is being sent.
-   */
-  async send(peerAddress: Address, msg: object) {
-    const modifiedMsg = { from: this.address, ...msg };
-    await this.messagingService.send(peerAddress, modifiedMsg);
-  }
-
-  // Note: The following getter/setter method will become private
-  // once the machine becomes embedded in the Node.
   /**
-   * Retrieves the value that's mapped to the given key through the provided
-   * When the given key doesn't map to any value, `null` is returned.
-   * store service.
-   * @param key
-   */
-  async get(key: string): Promise<any> {
-    return await this.storeService.get(key);
-  }
-
-  /**
-   * Sets the given value to the given key through the provided store service.
-   * @param key
-   * @param value
-   */
-  async set(key: string, value: any): Promise<any> {
-    return await this.storeService.set(key, value);
-  }
-
-  /**
-   * This sets up all the listeners for the methods the Node is expected to have
-   * as described at https://github.com/counterfactual/monorepo/blob/master/packages/cf.js/API_REFERENCE.md#node-protocol
-   *
-   * The responses to these calls are the events being listened on
-   * https://github.com/counterfactual/monorepo/blob/master/packages/cf.js/API_REFERENCE.md#events
+   * Enables making a direct call to the Node for a specific method instead
+   * of registering a callback for it.
```

Comment:
what does "instead of registering a callback" mean?

Reply:
As opposed to the `on` method above this which accepts a callback that can't be waited on

Maybe dropping the "instead of" is better:
```
   * Makes a direct call to the Node for a specific method.
```

Since the call is awaitable it'd be implied that the call is async, and does not necessarily need a callback.